gen-models/gen-models.capnp

```diff
@@ -130,9 +130,10 @@ struct Operation {
 }
 
 struct FileAddition {
-  id @0 :Int64;
+  id @0 :List(UInt8);
```

Comment:
I've been thinking some of the other IDs we have should probably get converted from eg i64 to u64